src/main/scala/io/github/pauljamescleary/petstore/domain/users/UserRepositoryAlgebra.scala

```diff
@@ -1,11 +1,13 @@
 package io.github.pauljamescleary.petstore.domain.users
 
+import cats.data.OptionT
+
 trait UserRepositoryAlgebra[F[_]] {
   def create(user: User): F[User]
 
   def update(user: User): F[Option[User]]
 
-  def get(userId: Long): F[Option[User]]
+  def get(userId: Long): OptionT[F, User]
```

Comment:
If we're making this change for this method, we should make the same change for the other two similar ones.

Reply:
I tried to make the lesser possible changes but I think this should be done to share a common codification and having OptionT is better if you ever want to change your monadic stack.

Reply:
BTW, there's no ListT but it can be achieved using Nested.